libraries/chain/transaction_context.cpp

```diff
@@ -173,7 +172,7 @@ namespace eosio { namespace chain {
          validate_account_cpu_usage( billed_cpu_time_us, validate_account_cpu_limit, true );
       }
 
-      eager_net_limit = (eager_net_limit/8)*8; // Round down to nearest multiple of word size (8 bytes) so check_net_usage can be efficient
+      //eager_net_limit = (eager_net_limit/8)*8; // Round down to nearest multiple of word size (8 bytes) so check_net_usage can be efficient
```

Comment:
This should not be removed.

Reply:
Correct, because it got rounded after /8 if eager_net_limit is capable to be separated with reminder it does, otherwise it does nothing. I beg your pardon, from first sight it can be seen like /8 *8 does nothing if eager_net_limit is divisible by 8 without reminder.